packages/genesys-spark-components/src/components/stable/gux-inline-alert/gux-inline-alert.tsx

```diff
@@ -50,10 +55,17 @@ export class GuxAlert {
           [`gux-${this.accent}`]: true
         }}
       >
-        <gux-icon icon-name={this.getIcon(this.accent)} decorative></gux-icon>
         <div class="gux-message-wrapper">
+          <gux-icon icon-name={this.getIcon(this.accent)} decorative></gux-icon>
           <div class="gux-sr-only">{this.i18n(this.accent)}</div>
-          <slot />
+          {hasSlot(this.root, 'emphasis-text') && (
+            <div class="gux-emphasis-text">
+              <slot name="emphasis-text" />
+            </div>
```

Comment:
I think this `emphasis-text` stuff is all coming from a misunderstanding of the example that was included with the original design.

There was nothing special about the `<strong>Note:</strong>` you can slot anything into the slot and emphasize it however you want.

Reply:
Its not just the alignment of the "Note text", its anything else you might put in there. For example, this code produces the image below:

```
<gux-inline-alert accent="error" style="width: 400px">
    <strong>Note:</strong> This is an alert which is wrapping to multiple lines. But it all falls apart of you add in a <a href="#">link</a> or any other formatting like <em>italics</em> or anything else.
  </gux-inline-alert>
```
  
  ![image](https://github.com/MyPureCloud/genesys-spark/assets/17025600/c6f65505-4eb2-42f1-be92-8db7c6b16b72)
  
  I'm not a fan of the new slot but I can't think of any other way to ensure the layout & styling.
  Open to suggestions.